tdrs-backend/tdpservice/search_indexes/admin/filter.py

```diff
@@ -0,0 +1,37 @@
+"""Filter classes."""
+from django.utils.translation import ugettext_lazy as _
+from django.contrib.admin import SimpleListFilter
+from tdpservice.data_files.models import DataFile
+
+class CreationDateFilter(SimpleListFilter):
+    """Simple filter class to show newest created datafile records."""
+
+    title = _('Newest')
+
+    parameter_name = 'created_at'
+
+    def lookups(self, request, model_admin):
+        """Available options in dropdown."""
+        return (
+            (None, _('Newest')),
```

Comment:
@elipe17 i only see this filter for TANF, not SSP. i think this is why im also seeing all versions in SSP but only the latest for TANF (when filter set to newest).

![2579p2](https://github.com/raft-tech/TANF-app/assets/63075587/6583def4-95ee-4e16-a5ec-3c174219a60f)
![2579p1](https://github.com/raft-tech/TANF-app/assets/63075587/14d8338a-244a-4aef-bf97-696cc596f2ef)